### DISCUSSION TITLE
Unstable branch broke(?) my database

### DISCUSSION THREAD

**Discussion by adamef93** [jellyfin/jellyfin#10362] [Q&A]
Not sure if this is intended behavior, but I switched to the unstable branch docker image to see if it resolved an issue I have with some subtitles (a constant {\an1} appearing) and now I can't revert back to the latest stable image without rebuilding my entire instance; losing watch history, up next, collections, the whole nine for myself and family/friends using my server. I really don't want to live on the unstable branch indefinitely even though things seem to be working for now and I also don't want to rebuild everything if at all possible. What are my options here?

**Comment by crobibero** [comment]
Your options are to restore from backup or start fresh. There is no support for downgrading.

**Comment by adamef93** [comment]
Awesome. Maybe put a note of that somewhere obvious.

**Comment by TheMelmacian** [comment]
You could wait till 10.9.0 is released before going back to the stable release. If non of the current changes of the unstable branch are changed in a breaking way they should become stable with the next major release and you can simply switch back to the stable image.
You only have to make sure not to upgrade your current unstable image further than a possible 10.9.0 release or you will have the same problem.

I can't recommend this but if you know you way around relational databases you could try to revert the database changes manually to make your current instance compatible again.

**Reply by adamef93** [reply]
Not a bad idea. I'll see if I can switch back once 10.9 hits GA
